Sources/MpcNET/Commands/Database/FindAndSearchCommands.cs

```diff
@@ -114,15 +379,31 @@ public class FindCommand : BaseFilterCommand
         /// <param name="windowEnd">End of the portion of the results desired</param>
         public FindCommand(List<KeyValuePair<ITag, string>> filters, int windowStart = -1, int windowEnd = -1) : base(filters, windowStart, windowEnd) { }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FindCommand"/> class.
+        /// </summary>
+        /// <param name="filter">Filter</param>
+        /// <param name="windowStart">Start of the portion of the results desired</param>
+        /// <param name="windowEnd">End of the portion of the results desired</param>
+        public FindCommand(IFilter filter, int windowStart = -1, int windowEnd = -1) : base(filter, windowStart, windowEnd) { }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FindCommand"/> class.
+        /// </summary>
+        /// <param name="filters">List of filters</param>
+        /// <param name="windowStart">Start of the portion of the results desired</param>
+        /// <param name="windowEnd">End of the portion of the results desired</param>
+        public FindCommand(List<IFilter> filters, int windowStart = -1, int windowEnd = -1) : base(filters, windowStart, windowEnd) { }
+
     }
 
-   
     /// <summary>
     /// Base class for find/search commands.
     /// </summary>
     public abstract class BaseFilterCommand : IMpcCommand<IEnumerable<IMpdFile>>
     {
-        private readonly List<KeyValuePair<ITag, string>> filters;
+        private readonly List<KeyValuePair<ITag, string>> tagFilters;
+        private readonly List<IFilter> completeFilters;
```

Comment:
```suggestion
        private readonly List<IFilter> _filters;
``` 

Keeps naming conventions in line with the other two private fields.  
(I'm aware the existing `filters` list didn't have the underscore, but I'm gonna blame that on the latest person who modified this class, which is probably me)